styles/docs.css

```diff
@@ -5,31 +5,108 @@
 .component-nav {
   top: 0;
   height: 100vh;
-}
-.component-nav-list {
+  width: 20%;
   overflow-y: auto;
-  height: 80vh;
+  overflow-x: hidden;
+  border-right: solid 1px var(--secondary-dark-color);
+}
+
+main {
+  width: 80%;
+}
+
+.component-nav ul li {
+  width: 100%;
+  padding: 1rem;
+}
+
+.component-nav-list li:hover {
+  background-color: var(--primary-faint-color);
+  color: var(--primary-color);
 }
+
+.component-nav-list li {
+  margin: 1rem 0;
+}
+
 .comp-nav-link-item {
-  color: var(--dark-text-color);
+  color: inherit;
+  padding: 1rem 12rem;
 }
-.comp-nav-link-item:hover {
+
+/* hamburger menu */
+.hamburger-icon {
   color: var(--primary-color);
 }
 
+#btn-hamburger {
+  display: none;
+}
+
+#nav-menu-hamburger {
+  border-bottom: solid 2px var(--secondary-dark-color);
+}
+
+.component-nav-hamburg {
+  height: 45vh;
```

Comment:
If there isn't any specific reason to set the height to 45vh, you should probably increase it to 100%.